### PR TITLE
feat(setup): add mui theme

### PR DESCRIPTION
**Pull request by MdRasulov**

### CODE REVIEW

**Review by fasbrothers — APPROVED** [review]

**Review by otabek — APPROVED** [review]

**Review by mirsadikov — APPROVED** [review]

File: src/index.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
```
Comment (gitkoshelew) [inline src/index.css]: use the global reset instead. There is normaization for all elements

Comment (gitkoshelew) [reply src/index.css]: https://v4.mui.com/ru/components/css-baseline/#global-reset

File: src/theme/index.ts [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    },
+  },
+  palette: {
+    primary: {
+      main: '#222222',
+    },
+    secondary: {
+      main: '#FF385C',
+    },
+    darkGrey: {
```
Comment (gitkoshelew) [inline src/theme/index.ts]: use more generic type name. What if this style color will be changed to green? better name is secondary2 or layout or mainText something like that

File: src/theme/index.ts [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+      'lg': 1024,
+      'xl': 1280,
+      '2xl': 1536,
+    },
+  },
+  components: {
+    MuiTypography: {
+      styleOverrides: {
+        root: {
+          color: '#222222',
```
Comment (gitkoshelew) [inline src/theme/index.ts]: create constant as soon you use the same value for palette primary and root color

**Review by gitkoshelew — CHANGES_REQUESTED** [review]

**Review by gitkoshelew — APPROVED** [review]

**Pull request merged**